src/controllers/RecommendationController.js

```diff
@@ -0,0 +1,177 @@
+import { Recommendation } from '../models';
+import { statusCodes, responseMessages } from '../constants';
+
+/**
+ * @description Re Controller class
+ */
+export default class RecommendationController {
+
+    /**
+     * Recommend a user
+     *
+     * @static
+     * @param {*} req
+     * @param {*} res
+     * @returns {object} recommendation
+     * @memberof RecommendationController
+     */
+    static async recommend(req, res) {
+        const { currentUser, receiver } = req;
+        const { description } = req.body;
+
+        if (currentUser._id.toString() === receiver.id) {
```

Comment:
According to the documentation, they advise to use `currentUser._id.equals(receiver.id)